src/Polly.Strategies.Bench/README.md

```diff
@@ -0,0 +1,40 @@
+﻿# Benchmark results
```

Comment:
Would be good to maybe refactor the benchmarks slightly to use different methods for v7 and v8 so we can use baseline and get the comparisons computed for us to make the tables, and this the improvements, easier to grok.

Reply:
Can you elaborate here? Not sure what to do and how to cahnge them.

Reply:
I meant something like this:

```csharp
public class IndexOfAnyBenchmarks
{
    private const string AUrlWithAPathAndQueryString = "http://www.example.com/path/to/file.html?query=string";
    private static readonly char[] QueryStringAndFragmentTokens = new[] { '?', '#' };

    [Benchmark(Baseline = true)]
    public int IndexOfAny_String()
    {
        return AUrlWithAPathAndQueryString.IndexOfAny(QueryStringAndFragmentTokens);
    }

    [Benchmark]
    public int IndexOfAny_Span_Array()
    {
        return AUrlWithAPathAndQueryString.AsSpan().IndexOfAny(QueryStringAndFragmentTokens);
    }
}
```

Then you get the extra columns in the results (`Ratio`, `RatioSD`, Alloc Ratio`) that show you how much faster `IndexOfAny_String` is compared to `IndexOfAny_Span_Array` in this example without having to work it out yourself (here, 29% faster).

``` ini

BenchmarkDotNet=v0.13.4, OS=Windows 11 (10.0.22000.1335/21H2)
Intel Core i9-9980HK CPU 2.40GHz, 1 CPU, 16 logical and 8 physical cores
.NET SDK=7.0.102
  [Host]     : .NET 6.0.13 (6.0.1322.58009), X64 RyuJIT AVX2
  DefaultJob : .NET 6.0.13 (6.0.1322.58009), X64 RyuJIT AVX2


```
|                    Method |     Mean |     Error |    StdDev |   Median | Ratio | RatioSD | Allocated | Alloc Ratio |
|-------------------------- |---------:|----------:|----------:|---------:|------:|--------:|----------:|------------:|
|         IndexOfAny_String | 9.167 ns | 0.2493 ns | 0.7193 ns | 8.938 ns |  1.00 |    0.00 |         - |          NA |
|     IndexOfAny_Span_Array | 6.529 ns | 0.1603 ns | 0.1574 ns | 6.525 ns |  0.71 |    0.03 |         - |          NA |

Reply:
Got it, updated the benchmarks slightly. Now baseline is also included.